R/dependencies.R

```diff
@@ -1709,8 +1709,18 @@ renv_dependencies_report <- function(errors) {
 
 renv_dependencies_eval <- function(expr) {
 
-  # create environment with small subset of symbols
-  syms <- c("list", "c")
+  # create environment with small subset of "safe" symbols, that
```

Comment:
In general, I'm not sure it's worth bothering to expand this since most calls will be like the example in the linked issue, i.e. `!use_cache`, where `use_cache` is computed in the doc.